code/modules/clothing/masks/gasmask.dm

```diff
@@ -109,6 +109,21 @@
 	flags_cover = MASKCOVERSEYES
 	burn_state = FLAMMABLE
 
+/obj/item/clothing/mask/gas/mime/attack_self(mob/user)
+

```

Comment:
This requires taking off your mask to change emotions.

Do alt+click.

Cycling would probably be more user friendly as well rather than dealing with a pop up